plugins/modules/tmp_vcenter_resourcepool_info.py

```diff
@@ -0,0 +1,43 @@
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+import ansible_collections.ansible.vmware_rest.plugins.module_utils.vmware_httpapi as vmware_httpapi
+
```

Comment:
This module is already covered in `vmware_core_info`

Reply:
Same answer.